tests/suite/zar/basic.yaml

```diff
@@ -16,11 +15,9 @@ inputs:
 outputs:
   - name: stdout
     data: |
-      #0:record[data_sort_direction:bool,log_size_threshold:float64,spans:array[record[log_id:string,span:record[dur:record[ns:float64,sec:float64],ts:record[ns:float64,sec:float64]]]],version:float64]
-      0:[F;2500;[[20200422/1587518620.0622373.zng;[[998533661;659;][63703640;1587517960;]]][20200422/1587517956.06264854.zng;[[992906871;802;][69741670;1587517153;]]][20200422/1587517152.06293072.zng;[[891391;726;][62039330;1587516426;]]][20200422/1587516416.06480216.zng;[[996562431;754;][68239730;1587515661;]]][20200422/1587515660.06416955.zng;[[995701351;834;][68468200;1587514825;]]][20200422/1587514825.06323209.zng;[[997709591;764;][65522500;1587514060;]]][20200422/1587514053.06464653.zng;[[995371641;836;][69274890;1587513216;]]][20200421/1587513216.06789438.zng;[[998046041;774;][69848340;1587512441;]]][20200421/1587512439.06556231.zng;[[996404251;692;][69158060;1587511746;]]][20200421/1587511734.06734765.zng;[[998719481;637;][68628170;1587511096;]]][20200421/1587511094.06170004.zng;[[999396291;672;][62303750;1587510421;]]][20200421/1587510414.06953991.zng;[[2232311;685;][67307600;1587509729;]]][20200421/1587509729.06618418.zng;[[3296581;877;][62887600;1587508852;]]][20200421/1587508850.06466032.zng;[[996137081;19;][68523240;1587508830;]]]]0;]
       ===
-      logs/20200422/1587518620.0622373.zng
-      logs/20200422/1587514053.06464653.zng
-      logs/20200421/1587513216.06789438.zng
+      20200422/1587518620.0622373.zng
+      20200422/1587514053.06464653.zng
+      20200421/1587513216.06789438.zng
```

Comment:
Nit: All these test changes appear to be cosmetic. If that's the case, don't include them in this PR.

Reply:
It is not (I def would not have gone through the trouble I know an other way). The uri change requires that paths get stored as absolute paths not relative paths. Thus left un-changed this would show the absolute path of a tempdir which would make the test fail everytime.

Reply:
to clarify: it's the new `data_path` field in zar.json that now contains an absolute path, not the individual entries for each log.

Reply:
@alfred-landrum is correct